### ISSUE TITLE
.Site.Author was deprecated in Hugo v0.124.0 and will be removed in a future release. Use taxonomies instead.

### ISSUE THREAD

**Issue by TheGroundZero** [rhazdon/hugo-theme-hello-friend-ng#481]
Warnings when building site with Hugo v0.134.0

```
WARN  deprecated: .Site.Author was deprecated in Hugo v0.124.0 and will be removed in a future release. Use taxonomies instead.
WARN  The author key in site configuration is deprecated. Use params.author.name instead.
```
https://github.com/rhazdon/hugo-theme-hello-friend-ng/blob/a902dacd8477728158ca181e17a340ea2f36d1ac/layouts/partials/footer.html#L6

https://github.com/rhazdon/hugo-theme-hello-friend-ng/blob/a902dacd8477728158ca181e17a340ea2f36d1ac/layouts/partials/head.html#L7

https://github.com/rhazdon/hugo-theme-hello-friend-ng/blob/a902dacd8477728158ca181e17a340ea2f36d1ac/layouts/posts/rss.xml#L7-L20

**Comment by stdevel** [comment]
Should be fixed with #493 - feel free to test 🙂

**Comment by TheGroundZero** [comment]
Fixed
